api/dps/server.go

```diff
@@ -80,9 +80,10 @@ func (s *Server) GetLast(_ context.Context, _ *GetLastRequest) (*GetLastResponse
 // GetHeight implements the `GetHeight` method of the generated GRPC
 // server.
 func (s *Server) GetHeightForBlock(_ context.Context, req *GetHeightForBlockRequest) (*GetHeightForBlockResponse, error) {
-
-	var blockID flow.Identifier
-	copy(blockID[:], req.BlockID)
+	if len(req.BlockID) != 32 {
+		return nil, fmt.Errorf("expecting %d bytes but got %d bytes", len(flow.Identifier{}), len(req.BlockID))
```

Comment:
Nitpick - it might be nice to be consistent with the general way of writing errors, which would usually be something like `unexpected length (have: %d, want: %d)`.

Or at least I think we were doing that, we might have mixed it up lately :D

Reply:
I copied this from the error handling Flow uses, since this is inevitably what will come up on our side as well for Paths for example (since we use their func to convert them, which does this check). So basically I just wanted it to be consistent between StateCommitments, ledger.Paths and flow.Identifiers 😄